packages/gatsby/src/bootstrap/index.js

```diff
@@ -385,7 +386,16 @@ module.exports = async (args: BootstrapArgs) => {
 
   const graphqlRunner = (query, context = {}) => {
     const schema = store.getState().schema
-    return graphql(schema, query, context, context, context)
+    return graphql(
+      schema,
+      query,
+      context,
+      {
+        ...context,
+        nodeModel: nodeStore,
```

Comment:
If we expose the store directly, should users register page dependencies with getNodeAndSavePathDependency?